### PR TITLE
changed order of mini fabs

### PR DESCRIPTION
**Pull request by AndyScherzinger**
for details please see #1581 

pinging @davivel

### CODE REVIEW

File: res/layout/list_fragment.xml [hunk 1]
```diff
@@ -113,21 +113,21 @@
...
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             fab:fab_size="mini"
-            fab:fab_icon="@drawable/ic_action_create_dir"
+            fab:fab_icon="@drawable/ic_import"
             fab:fab_colorNormal="@color/primary_button_background_color"
             fab:fab_colorPressed="@color/owncloud_blue"
             fab:fab_title=""/>
-
+
```
Comment (tobiasKaminsky) [inline res/layout/list_fragment.xml]: unnecessary change ;)

**Comment by tobiasKaminsky** [comment]
Code is fine for me :+1:

File: res/layout/list_fragment.xml (hunk 1 above)
Comment (AndyScherzinger) [inline res/layout/list_fragment.xml]: Yeah, did the move in a text pad :D

**Comment by davivel** [comment]
Cool! 

Just one thing. Since this was listed as a bug in regression tests (not completely true, I know), could you make the PR to branch release-2?

Or even, just cherry-pick the commit to branch release-2 and push it there. 

As a general rule, we fix bugs found during regression testing in the corresponding release branch directly instead of master.

**Comment by AndyScherzinger** [comment]
as discussed, directly pushed to releaxse-2 branch

File: res/layout/list_fragment.xml (hunk 1 above)
Comment (AndyScherzinger) [inline res/layout/list_fragment.xml]: Re-did it on release-2, with absolute minimal change :D https://github.com/owncloud/android/commit/2a2eceed8db0959375e40510a256cbeeeef1c09d

**Comment by davivel** [comment]
Thanks a lot @AndyScherzinger

**Comment by AndyScherzinger** [comment]
You are welcome! Looking forward to the release! :)
